types/jquery.contextmenu/index.d.ts

```diff
@@ -22,8 +22,8 @@ interface JQueryContextMenuOptions {
         hide?: string;
     };
     events?: {
-        show?: () => void;
-        hide?: () => void;
+        show?: (options?: any) => any;
+        hide?: (options?: any) => any;
```

Comment:
As discussed [here](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html), it's not recommended to have optional parameters in callbacks. It looks like the framework will always call this method with the options parameter, so it should be required.